src/neuroconv/datainterfaces/ophys/cnmfe/cnmfedatainterface.py

```diff
@@ -3,7 +3,7 @@
 
 
 class CnmfeSegmentationInterface(BaseSegmentationExtractorInterface):
-    """Data interface for constrained nonnegative matrix factorization (CNMFE) segmentation extractor."""
+    """Data interface for convolutional non-negative matrix factorization (CNMFE) segmentation extractor."""
```

Comment:
I thgouht that it stod forconstrained (not convolutional). I got it from this paper:
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5871355/

Reply:
```suggestion
    """Data interface for constrained non-negative matrix factorization (CNMFE) segmentation extractor."""
```

Reply:
huh, I always thought it was convolutional!